ror-demo-cluster/conf/es/readonlyrest.yml

```
@@ -37,6 +37,27 @@ readonlyrest:
         access: rw
         hide_apps: ["Security", "Observability"]
 
+    # Requirement B (solution 1.)
+    - name: "Allow RW access to all tracy-* indices"
+      groups_or: ["kib_tracy_default_adm", "kib_tracy_tenant1_adm"]
+      indices: ["deva-*", "int-*", "prs-*", "pred-*", "tracy-*"]
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)